public/keymaps/c/clawsome_roadster_default.json

```
@@ -0,0 +1,21 @@
+{
+  "keyboard": "clawsome/roadster",
+  "keymap": "default",
+  "commit": "14cb662280852f6cd4d84dae9e46c00fdd971ef4",
+  "layout": "LAYOUT",
+  "layers": [
+    [
+      "KC_ESC",  "KC_Q",    "KC_W",    "KC_E",    "KC_R",    "KC_T",    "KC_Y",    "KC_U",    "KC_I",    "KC_O",    "KC_P",    "KC_DEL",
+      "KC_TAB",  "KC_A",    "KC_S",    "KC_D",    "KC_F",    "KC_G",    "KC_H",    "KC_J",    "KC_K",    "KC_L",    "KC_ENT",
+      "KC_LSFT", "KC_FN"    "KC_Z",    "KC_X",    "KC_C",    "KC_V",    "KC_B",    "KC_N",    "KC_M",    "KC_DOT",  "KC_UP",   "KC_RSFT",
+      "KC_LCTL", "KC_LGUI", "KC_LALT",            "KC_SPC"   "KC_SPC",  "KC_SPC"                         "KC_LEFT", "KC_DOWN", "KC_RGHT",
```

Fixing some syntax errors I missed on the initial review.

```suggestion
      "KC_LSFT", "XXXXXXX", "KC_Z",    "KC_X",    "KC_C",    "KC_V",    "KC_B",    "KC_N",    "KC_M",    "KC_DOT",  "KC_UP",   "KC_RSFT",
      "KC_LCTL", "KC_LGUI", "KC_LALT",            "KC_SPC",  "KC_SPC",  "KC_SPC",                        "KC_LEFT", "KC_DOWN", "KC_RGHT"
```